docs/_blog/2025-09-01/rocrate-at-senckenberg.md

```diff
@@ -0,0 +1,96 @@
+enriched with contextual information and provenance data.
+To make these datasets discoverable, we combine the approach
+of serving detached RO-Crates with FAIR signposting on our platform,
+making it easy for machine agents to discover relevant metadata for automated processing.
+
+## workflow.earth: Computational Reproducibility in Earth Observation
+
+Our FAIR Workflow Platform [workflow.earth](https://workflow.earth),
+deployed within the [Destination Earth](http://destination-earth.eu/) Data Lake (DEDL),
+tackles a different challenge:
+making computational workflows and their outputs FAIR
+from execution through to publication.
+Within the platform, researchers can submit workflows as executable Workflow RO-Crates.
+The platform executes the workflows and automatically generates
+FAIR-compliant datasets from workflow execution,
+thus eliminating the manual overhead associated with FAIR data publication.
+Execution metadata, parameters, authorship, etc. are all automatically captured
+and made available as a single public Workflow Run RO-Crate
+with the aim of enabling seamless data movement between repositories and data spaces.
+
+This platform currently powers BioDT’s [Crop Wild Relatives Digital Twin](https://biodt.eu/use-cases/crop-wild-relatives),
+which models habitat suitability for wild plant relatives of crops
+to develop climate-resilient agriculture.
+By packaging the entire computational process in RO-Crate,
+we ensure that this climate adaptation research can be reproduced,
+validated, and built upon by the global research community.
+
+## Why RO-Crates work for us
+
+Through our projects, we've learned that
+applying the FAIR principles to research requires more than technical standards:
+it requires the existence of working implementations,
+both for showcasing the benefits of these standards,
+and for establishing an ecosystem where it makes sense to adapt these technologies in the first place.
+
+RO-Crates succeed in this context for us because
+they strike the right balance between simplicity and practical usability.
+The foundation in JSON-LD makes them approachable for researchers without
+deep technical expertise in ontologies, while the extensibility enables
+adopting them to address real pain points researchers face
+when it comes to research data management.
+Finally, the growing ecosystem of tools such as [ro-crate-py](https://github.com/ResearchObject/ro-crate-py)
+and platforms such as [WorkflowHub](https://workflowhub.eu/) means
+researchers can find support for their implementation needs.
+Through our contributions, we aim to strengthen this ecosystem
+and enable researchers and other infrastructures to adopt the FAIR principles.
+
+## Resources
+
+Our work has produced several open-source contributions to the RO-Crate ecosystem.
+Additionally, a lot of our developed software is available open source
+at our [Github organization](https://github.com/Senckenberg-DCBiodivIT).
```

Comment:
You could include a contact email for yourself/your team at the end of the post, in case anyone wants to discuss these topics with you further. (optional)